src/backend/db/products.js

```diff
@@ -10,6 +10,7 @@ export const products = [
     _id: uuid(),
     title: "Immortals of Meluha",
     author: "Amish",
+    rating: "5",
     mrpPrice: "499",
     discountPrice: "476",
```

Comment:
Suggestion, but you dont have to explicitly specify the discount amount inside your mockAPI,
you already have the base price and the discount %, so why not calculate the final price after discount in the component itself?

Reply:
I did it so that on client side it didn't had to do all the calculations, but I understand your concern, I'll look into it and introspect more!